### DISCUSSION TITLE
Home Pro Current Demand Electricity out by factor of 10

### DISCUSSION THREAD

**Discussion by craigarthur** [BottlecapDave/HomeAssistant-OctopusEnergy#998] [Q&A]
I've recently hooked up a home pro to a home assistant instance i've just set up.  I've got the data all flowing through nicely, but one thing I've noticed is that the "Current Demand Electricity" levels are out by a factor of 10 v's the mini's readings.  Current usage is showing as 5,282.0W instead of 528.20W

What would i need to edit to make that number accurate?  Everyhting else seems to be correct and working well as best as I can tell.

**Comment by BottlecapDave** [comment]
Hi, sorry for the late response. The sensor just reports the value returned by the API which in turn just returns the value returned by the raw API on the Home Pro device.

If you run the script on your home pro device, as per https://github.com/OctopusSmartEnergy/Home-Pro-SDK-Public/blob/main/Home.md#octave_energy, are the values aligned? Take into account the home pro script returns data in kilowatts vs the HA sensor which reports in watts.

**Reply by craigarthur** [reply]
It was, I tweaked the followin:

 consumption = float(meter_consump['consum']['consumption'])/10000.0
    instdmand = float(meter_consump['consum']['instdmand'])/10000.0

to change from 1000, to 10000 and it displayes the same as the Mini now.  Will see if there is a way of tweaking the HA-EO script.

**Reply by BottlecapDave** [reply]
This is probably something to raise with AgilePhil in the OE forums as it might be effecting other users or be specific to your meter.

**Comment by aioue** [comment]
Dropping a comment for other viewers - Craig is talking about the "Ohme Home Pro" EV charger, not an upgraded Home Mini device you weren't aware of 🙂
